maxim/logger/bedrock/client.py

```
@@ -218,15 +265,24 @@ def converse(
         if performanceConfig:
             api_params["performanceConfig"] = performanceConfig
         if additionalModelRequestFields:
-             api_params["additionalModelRequestFields"] = additionalModelRequestFields        
+            api_params["additionalModelRequestFields"] = additionalModelRequestFields
         response = self._client.converse(**api_params)
         changed_response = BedrockUtils.parse_message(response)        
```

Flake8 (7.2.0)

[warning] trailing whitespace

(W291)